_posts/2020-11-04-nowa-forma-meetupu.md

```diff
@@ -0,0 +1,44 @@
+---
+layout:    post
+title:     "Consdata Tech Webinar - rozmowa z Marcinem Mergo o nowej formie meetupu"
+date:      2020-11-04 08:00:00 +0100
+published: true
+author:    mmergo
```

Comment:
to może adam powinien być autorem wpisu skoro i tak jest pod nim podpisany w treści? :)

Reply:
@adampij, Ty miałeś tutaj jakiś pomysł? :)

Reply:
Rozważaliśmy za i przeciw takiego podejścia, ale Marcin stanowi tutaj pewnie z 95% całości i stąd lepiej żeby to Marcin widniał jako autor. Ufamy też, że Marcin będzie dużo większym magnesem niż ja, zwłaszcza że większy szperacz znajdzie bez problemu i wcześniejszy wpis Marcina o CDTech. To nie przez skromność nie wyciągam siebie na wierzch :)

Podpisałem się między innymi po to by nie wyglądało na to, że Marcin sam sobie zadał pytania i potem na nie odpowiedział.

Reply:
skoro obaj uważacie, że tak jest najlepiej